kong/plugins/file-log/handler.lua

```diff
@@ -69,6 +73,12 @@ local FileLogHandler = {
 
 
 function FileLogHandler:log(conf)
+  if conf.custom_fields_by_lua then
+    for key, expression in pairs(conf.custom_fields_by_lua) do
```

Comment:
Slight concern on this usage of `pairs` inside a hot path. It can not be JIT compiled by LuaJIT and may result in slow execution. Any chance `pairs` can be avoided here?

It's not a blocker for the PR, but just making a note here so we remember to look into it.

Reply:
I will look into this after the beta.

Reply:
the `pairs` loop goes around a `sandbox(expression, ...)`; i don't think it could be considered a tight loop.

tangentially related:  `pairs()` isn't an NYI because it's too complex, but because the interpreted version is faster in most cases.  the downside is that it breaks surrounding traces, which is bad if there's tight code with high interpretation penalty (like FFI).   for "wide" loops it's ok, the result will be a few more chained sidetraces.